lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -308,7 +327,7 @@ const Reportback = {
         'isListItem': true,  // @TODO: probably a better way to address this... :insert shruggy emoji here:
         'reactions': {
           enabled: setting('dsKudosReactions.enabled'),
-          data: this.getReactions(items[i].kudos.data),
+          data: setting('dsKudosReactions.likedByCurrentUser') ? this.getReactions(items[i].kudos, true) : this.getReactions(items[i].kudos.data, false),
         }
```

Comment:
Ideally, we'd want to avoid doing any sort of conditional logic here and just return an "identical" response from Rogue. Is it possible to manipulate Rogue's reportback endpoint so it has `kudos.data` structure too?